### DISCUSSION TITLE
Feature Request - Zone editor within the configuration

### DISCUSSION THREAD

**Discussion by Tipalo** [ufodone/envisalink_new#82] [Ideas]
Greetings,
Thanks for the nice integration, glad to see something that has zone bypasses included.

Once the zones have been created there was no visible means to edit or delete "spare" zones.  It would be nice to see this in the configuration.

Might also consider having Spare as a Zone definition as well for those that want them listed. A unique icon rather than the Opening icon would be good.

Thanks

**Comment by ufodone** [comment]
If I  understand you correctly, you have zones showing up that your alarm panel doesn't use.  If this is the case, you can specify exactly which zones you want created in the "Zone list" configuration setting.  For example, `1-4,9-10,13,15` would result in only zones 1, 2, 3, 4, 9, 10, 13, and 15 being created.  If you add additional zone usage in the future you can go back and edit this to include any newly added (or removed) zones.
